package.json

```diff
@@ -26,8 +26,8 @@
     "build:typing": "tsc --emitDeclarationOnly --outDir ./dist",
     "build:umd": "./scripts/build-webpack.sh",
     "build:umd_windows": ".\\scripts\\build-webpack.bat",
-    "build:copy": "cp README.md dist && cp package.json dist && cp LICENSE dist",
-    "build:copy_windows": "copy README.md dist && copy package.json dist && copy LICENSE dist",
+    "build:copy": "cp README.md dist && cp package.json dist && cp LICENSE dist && cp -R src dist/src",
+    "build:copy_windows": "copy README.md dist && copy package.json dist && copy LICENSE dist && xcopy .\\src .\\dist\\src",
```

Comment:
"build:copy_windows": "copy README.md dist && copy package.json dist && copy LICENSE dist && xcopy .\\src .\\dist\\src  **/S/I/Y**"

benutzen, sonst kopiert er einige Unterverzeichnisse nicht mit. Ausserdem kommt sonst immer eine Abfrage nach Überschreiben.

Reply:
updated, thank you I learned new thing .. and German language 😅